src/components/Friend/EditMemberComponent.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { Alert, StyleSheet } from "react-native";
+import { Button, ListItem } from "react-native-elements";
+import { firebase } from "../../firebase/Config";
+import {
+  cancelGroupInvitation,
+  removeUserFromGroup
+} from "../../firebase/HandleGroup";
+import { contactType } from "../../constants/Contact";
+import ContactBar from "./ContactBar";
+
+const EditMemberComponent = ({ items, isMember }) => {
+  const userID = firebase.auth().currentUser.uid;
+  let EditButton;
+
+  if (isMember) {
+    EditButton = ({ item }) => (
+      <Button
+        title="Remove"
```

Comment:
From What I observe, the resulting button only differs in the Alert tittle and the alert onPress event. So, it might be good to split the cases for the onPress listenet and the Alert title instead. For example, you can do

`alertTitle = isMember? "1stCase" : "2ndCase" ;
onPress = isMember? 1stListener: 2ndListener`

and then construct the edit button accordingly